src/components/About/About.test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { About } from "./About";
+
+describe("Test the About page", () => {
+  test("renders the About section", () => {
+    render(
+      <MemoryRouter>
+        <About />
+      </MemoryRouter>
+    );
+
+    const linkElement = screen.getByText(/About CAM API/i);
+    expect(linkElement).toBeInTheDocument();
+
+    const btns = screen.getAllByText("Request an API Key");
+    expect(btns).toBeDefined();
+  });
+
+  test("renders the CAMDP section", () => {
+    render(
+      <MemoryRouter>
+        <About />
+      </MemoryRouter>
+    );
+
+    const linkElement = screen.getByText(/Clean Air Markets Program Data/i);
+    expect(linkElement).toBeInTheDocument();
+
+    const btns = screen.getAllByText("CAMPD Application");
+    expect(btns).toBeDefined();
+  });
+
+  test("renders the ECMPS 2.0 section", () => {
+    render(
+      <MemoryRouter>
+        <About />
+      </MemoryRouter>
+    );
+
+    const linkElement = screen.getByText(/ECMPS 2.0 Beta/i);
+    expect(linkElement).toBeInTheDocument();
+
+    const btns = screen.getAllByText("ECMPS 2.0 Application");
```

Comment:
Is there a way we can test the button works by using [this](https://testing-library.com/docs/react-testing-library/example-intro#act)?

Reply:
@j-tafoya perhaps! Heads up @ntknguyen, I am finishing the existing tests to cover the paragraph text, I'll push the changes and we can look into @j-tafoya suggestion above.

Reply:
@j-tafoya the tests now utilize the fireEvent method. I commented out the old test for now but can delete it whenever. Lmk if this all looks good, thx.